app/viewmodels/SummaryCardHelper.scala

```diff
@@ -325,9 +325,14 @@ class SummaryCardHelper @Inject()(link: views.html.components.link) extends Impl
         case (_, _, _) => renderTag(messages("status.active")) // Temp solution
       }
     } else {
-      lpp.get.status match {
-        case PointStatusEnum.Paid => renderTag(messages("status.paid"))
-        case _ => renderTag(messages("status.due"), "penalty-due-tag")
+      val latePaymentPenaltyStatus = lpp.get.status
+      val latePaymentPenaltyAppealStatus = lpp.get.appealStatus
+
+      (latePaymentPenaltyAppealStatus,latePaymentPenaltyStatus) match {
+        case (Some(AppealStatusEnum.Accepted),_) => renderTag(messages("status.cancelled"))
+        case (Some(AppealStatusEnum.Reinstated),_) => renderTag(messages("status.due"), "penalty-due-tag")
+        case (_,PointStatusEnum.Paid) => renderTag(messages("status.paid"))
```

Comment:
The precedence will need to be changed here. If the penalty has been PAID then that should be above the REINSTATED case statement, if you wanted you could remove `case (Some(AppealStatusEnum.Reinstated),_) => renderTag(messages("status.due"), "penalty-due-tag")` and let it fall into the `case (_,_)` statement.